docs/content/stable/additional-features/pg-extensions/extension-pgparquet.md

```diff
@@ -0,0 +1,334 @@
+---
+title: pg_parquet extension
+headerTitle: pg_parquet extension
+linkTitle: pg_parquet
+description: Using the pg_parquet extension in YugabyteDB
+tags:
+  feature: early-access
+menu:
+  preview:
+    identifier: extension-pgparquet
+    parent: pg-extensions
+    weight: 20
+type: docs
+---
+
+The [pg_parquet](https://www.crunchydata.com/blog/pg_parquet-an-extension-to-connect-postgres-and-parquet) extension allows you to read from and write to [Parquet](https://parquet.apache.org/) files by extending the PostgreSQL COPY command. Files can be located in S3, Azure Blob Storage, Google Cloud Storage, http(s) endpoints, or on a file system.
+
+## Enable pg_parquet
+
+To enable the pg_parquet extension:
+
+1. Add `pg_parquet` to `shared_preload_libraries` in the PostgreSQL server configuration parameters using the YB-TServer [--ysql_pg_conf_csv](../../../reference/configuration/yb-tserver/#ysql-pg-conf-csv) flag:
```

Comment:
Why is this necessary? We don't have this step with other "pre-bundled" extensions